aui.views/src/AUI/ASS/Selector/CombinedSelector.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * AUI Framework - Declarative UI toolkit for modern C++20
+ * Copyright (C) 2020-2024 Alex2772 and Contributors
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include "AAssSelector.h"
+
+namespace ass {
+    template <typename L, typename R>
+    struct CombinedSelector : public IAssSubSelector {
```

Comment:
Since it acts like `AND` operator, I'd like to suggest `AndSelector` for this struct

Reply:
Done